chart/templates/platform-rbac.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if include "vcluster.rbac.createPlatformSecretRole" . }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: vcluster-platform-secret-role
```

Comment:
This needs to be dynamic based on the vcluster name as this would otherwise collide if there are multiple vclusters

Reply:
Same goes for the role binding